versions/v1.0.0/Social/Delete/main.adoc

```diff
@@ -24,6 +24,22 @@ interface Delete {
 }
 ----
 
+===== Fields
+
+`handle` *Optional*:: Handle/username of the user who deleted the post, e.g. `ordi.csb`.
```

Comment:
What does Handle/username mean? why is handle used as an alternative to username?

Reply:
`handle` is a specific username in some platform like lens, so we keep both.